packages/proxy/schema/index.ts

```diff
@@ -28,6 +29,7 @@ export type ModelEndpointType = (typeof ModelEndpointType)[number];
 export interface ModelSpec {
   format: ModelFormat;
   flavor: PromptInputType;
+  multimodal?: boolean;
```

Comment:
We should update `BaseMetadataSchema` in proxy/schema/secret.ts as well, so that users can input multimodal models into their list